src/include/cell_inline.h

```diff
@@ -845,6 +866,19 @@ __wt_cell_unpack_safe(WT_SESSION_IMPL *session, const WT_PAGE_HEADER *dsk, WT_CE
         break;
     }
 
+    /* Unpack any fast-truncate information. */
+    if (unpack->raw == WT_CELL_ADDR_DEL && F_ISSET(dsk, WT_PAGE_FT_UPDATE)) {
+        page_del = &unpack_addr->page_del;
+        WT_RET(__wt_vunpack_uint(
```

Comment:
This `page_del->txnid` needs same time window clear logic in `__cell_addr_window_cleanup`. Otherwise, we will read a stale transaction id upon restart.

Reply:
Good catch; I thought about that but incorrectly decided it wasn't needed because it's a stop time. (The cell-unpacking window cleanup doesn't touch stop times; but it does touch stop txnids.)